ci/cifailurejob.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: "container-pipeline-ci-job"
```

Comment:
Should we name it `container-pipeline-failure-ci-job` since this is checking for failure case(s)?

Reply:
I missed this while separating the success and failure jobs, updated.